packages/spaces/src/SpacesLink.js

```diff
@@ -280,6 +288,7 @@ Link.propTypes = {
   skeletonProps: skeletonPropType,
   maxDescriptionLength: PropTypes.number,
   linkAttributes: PropTypes.object,
+  role: PropTypes.string,
 };
 
 Link.defaultProps = {
```

Comment:
We should move from `Component.defaultProps` to default assignments, rest LGTM though
https://github.com/facebook/react/pull/16210
https://github.com/yannickcr/eslint-plugin-react/issues/2396#issuecomment-539184761

Reply:
Yeah, I've also got a ticket to convert to Typescript in this epic. So, all this prop-related stuff will be cleaned up.